src/interpreter/inner.rs

```diff
@@ -211,7 +211,7 @@ pub(super) fn from_txdata<'txin>(
                 // Key spend
                 // let sig =
                 // Tr inference to be done in future commit
-                todo!();
+                panic!("TODO");
```

Comment:
In 277c78392be39bb2755cd00fe6168fb2de7e685c:

lol does this change anything?

Reply:
Nope. Just allows compiling on MSRV :P . I was testing whether that particular commit compiles on rust 1.29